### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Any
+
+
+def cache(func: Any) -> Any:
```
Comment (EdAlekseiev) [inline app/main.py:4]:
```suggestion
def cache(func: Callable) -> Callable:
```

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    cache_update = {}
+
+    def wrapper(*args: Any) -> Any:
```
Comment (EdAlekseiev) [inline app/main.py:7]:
```suggestion
    def wrapper(*args) -> Any:
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Any, Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_update = {}
+
+    def wrapper(*args) -> Any:
```
Comment (olena-hul) [inline app/main.py:7]: Add annotation for args

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**